meta-lmp-base/recipes-core/initrdscripts/initramfs-framework/cryptfs

```diff
@@ -13,7 +13,7 @@ cryptfs_enabled() {
 
 e2fsck_check() {
 	if [ -n "`which e2fsck`" ]; then
-		fsckout=`e2fsck -p -v ${1}`
+		fsckout=`e2fsck -p -v -f ${1}`
```

Comment:
Here it will cause every boot to run with a forced fsck, while we only really need before resizing.

Can you change to accept a parameter and only call with -f on the block that performs the resizing?